src/docfx/lib/github/GitHubUserCache.cs

```diff
@@ -236,15 +252,21 @@ private DateTime NextExpiry()
 
         private async Task ReadCacheFile()
         {
-            if (File.Exists(_cachePath))
+            await Read(_cachePath);
+            await Read(_restorePath);
```

Comment:
>_restorePath [](start = 23, length = 12)

first restorePath and then cachePath? it's overwrite right?

Reply:
This is more like merge. I think the order does not matter in most cases.

---
In reply to: [239724827](https://github.com/dotnet/docfx/pull/3740#discussion_r239724827) [](ancestors = 239724827)